src/stories/WhyAndWhen.stories.mdx

```diff
@@ -157,6 +157,16 @@ We will use **Grab my IP** solution as an example.
       >
         This link is a real live solution using this approach.
       </a>
+      <br />
```

Comment:
Just to complete this example, can you make this into a custom component as its 'everywhere'